Go Client/Go Rules.cpp

```diff
@@ -16,17 +16,20 @@ void Rules::DoRules(stone StoneChecking, GoBoard &goBoard)
 			checkedCoordinates = {};
```

Comment:
this whole block should really move to a function... 3 levels of nesting inside a function is a wee bit deep